### PR TITLE
Added my solution and a time test

### PR DESCRIPTION
**Pull request by carmocca**
My solution to #13

### CODE REVIEW

**Comment by tonilopezmr** [comment]
Well @carmocca you don't have a CarlosImp because you've deleted!

 *Why?*

**Comment by carmocca** [comment]
Sorry about that, i've fixed it now and added the files.
This pull now adds my implementation and a class called numberOfMarksCost that lets you choose which class do you want to test.

**Comment by tonilopezmr** [comment]
only need to do one thing, the 3º point in issue #13 

In class ExamMarksFactory add your implementation CarlosImp in create() method and you will can see test passes in test package.

Use you Intellij Idea to open the project?

**Comment by carmocca** [comment]
Hi, sorry for all these commits :sweat_smile:, ill try to be cleaner next time. Changes:

- Fixed CarlosImp, it didn't pass the tests before.
- Fixed typo Qeue.
- Added NumberOfMarksCost, a class that compares our implementations speed (It used reflection before, now it uses Factory) i also moved it from src to tests.
- Added my implementation to ExamMarksFactory.

I believe everything is working properly now. If you think everything is correct, feel free to merge.

File: java/test/problems/problem2/NumberOfMarksCost.java [hunk 1]
```diff
@@ -25,35 +23,34 @@
...
-            System.out.println("\n#---------------------------------------------");
-            System.out.println("#         Measurement of search times: ");
-            System.out.println("#          Marks from 0 to "+MAX_ARRAY+"    ");
-            System.out.println("#--------------------------------------------- ");
-            System.out.println("#    Size    Search time   Nº Marks > "+MARK+"   ");
-            System.out.println("#---------------------------------------------");
+        int [] vector;
+        double t1, t2, time;
+        t1 = t2 = time = 0;
+        List<ExamMarks> list = new ExamMarksFactory().create();
```
Comment (tonilopezmr) [inline java/test/problems/problem2/NumberOfMarksCost.java]: Wow amazing!! You can see that ExamMarksFactory is very powerful!!!

**Comment by tonilopezmr** [comment]
No No!!! all commits is necessary!! 

It's better a lots of commits but clear and little than one big 

Thanks you guyy!!

**Pull request merged**